# Detector Settings
revision_name: latest
classifier_settings_postfix: sim

camera_topic: /bottom_camera/rgb/image_raw

# Debug settings
publish_visualization: true
# Boundary SVM's C parameters
boundary_svm_c: 0.025

# Training settings
# Filter settings which are saved with classifier settings objects
# Target size of resized image
target_width: 152
target_height: 122

# Minimum detection height (used for scaling images)
min_height: 0.7

# Filter settings
kernel_size: 5
sigmas: [0.2, 0.4, 0.6]
num_orientations: 6
stride: 3
average_size: 10

# Floor classifier parameters (RBF kernel)
train_c: 25.0
train_gamma: 2.0

# Training settings
floor_train_start_image: 0
floor_train_images: 200
antifloor_train_start_image: 0
antifloor_train_images: 200
